task2/script.js

```diff
@@ -0,0 +1,17 @@
+const liElement = document.createElement('li');
+
+const anchorElement = document.createElement('a');
+anchorElement.setAttribute('href', '1.html');
+anchorElement.textContent = 'これです';
```

Comment:
ref `innerHTML vs textContent`
https://developer.mozilla.org/ja/docs/Web/API/Node/textContent#differences_from_innerhtml

Reply:
> [IMO]
> I think it will be easier to understand if the variable name is written more simply.
> Example:) anchorElement → a

I understand.
I would like to change it that way.

> ref `innerHTML vs textContent`

I read the references you sent me.
It's not perfect, but I think you understand it.

Thank you for your advice